### DISCUSSION TITLE
Missing District,Tehsil identification and small villages names

### DISCUSSION THREAD

**Discussion by vishu-3** [dr5hn/countries-states-cities-database#273] [Ideas]
Ur API is good but lack the information to identify the district,tehsils and missing names of small villages.I'm from India.Is there any API provided by the Indian Government to categorize states,district,tehsil and villages accordingly.I've tried to check the data of data.gov.in but its little bit difficult to locate data.

**Comment by dr5hn** [comment]
@vishu-3 I found few APIs maybe it will help to fulfill your purpose.

http://www.postalpincode.in/Api-Details
https://rapidapi.com/vigowebs/api/india-pincode-with-latitude-and-longitude

**Comment by vishu-3** [comment]
Hello Darshan,
     Thanks for the help and the api provided by u.I'll check it.
  Thank You...

Yours Frnd
Vishal Upare
